test/parallel/test-assert.js

```diff
@@ -631,13 +631,13 @@ common.expectsError(
   }
 );
 common.expectsError(
-  () => assert(typeof 123 === 'string'),
+  () => assert(typeof 123n === 'string'),
   {
     code: 'ERR_ASSERTION',
     type: assert.AssertionError,
     generatedMessage: true,
     message: 'The expression evaluated to a falsy value:\n\n  ' +
-             "assert(typeof 123 === 'string')\n"
+             "assert(typeof 123n === 'string')\n"
```

Comment:
This adds support for multiple new features beyond just big int support but those aren't included in the tests. May be worthwhile adding additional tests for those things if possible?

Reply:
I added tests that verify private properties, class members and static class members. It is only verified in the REPL but I guess we do not have to verify that it works in each part. We can not yet test for the numeric separators, so we have to add tests as soon as V8 supports that.